mirdata/datasets/haydn_op20.py

```diff
@@ -0,0 +1,328 @@
+)
+
+
+class Track(core.Track):
+    """haydn op20 track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        title (str): title of the track
+        track_id (str): track id
+        humdrum_annotated_path (str): path to humdrum annotated score
+
+    Cached Properties:
+        key (str): key annotation
+        score (np.array): computed audio spectrum
+        keys (list): annotated local key
+        roman_numerals (list): MusicBrainz metadata
+        chords (list): annotated local key
+        duration (int): relative duration
+        midi_path (str): path to midi
+        score (music21.stream.Score): music21 score
+    """
+    def __init__(
+        self,
+        track_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            track_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+        self.humdrum_annotated_path = os.path.join(self._data_home, self._track_paths["annotations"][0])
+        self.title = os.path.splitext(self._track_paths["annotations"][0])[0]
+
+    def show(self):
+        show_score(self.humdrum_annotated_path)
+
+    @core.cached_property
+    def score(self) -> music21.stream.Score:
+        return load_score(self.humdrum_annotated_path)
+
+    @core.cached_property
+    def keys(self, resolution=28) -> List[dict]:
+        return load_key(self.humdrum_annotated_path, resolution)
+
+    @core.cached_property
+    def roman_numerals(self, resolution=28) -> List[dict]:
+        return load_roman_numerals(self.humdrum_annotated_path, resolution)
+
+    @core.cached_property
+    def chords(self, resolution=28) -> List[dict]:
+        return load_roman_numerals(self.humdrum_annotated_path, resolution)
+
+    @core.cached_property
+    def duration(self) -> int:
+        return self.chords[-1]['time']
+
+    @core.cached_property
+    def midi_path(self) -> str:
+        return load_midi_path(self.humdrum_annotated_path)
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            metadata={
+                "duration": self.duration,
+                "title": self.title,
+                "keys": self.keys,
+                "roman_numerals": self.roman_numerals,
+                "midi_path": self.midi_path,
+                "score": self.score,
+                "humdrum_annotated_path": self.humdrum_annotated_path
+            },
+        )
+
+
+def load_score(path):
+    """Load haydn op20 score from a file with music21 format (music21.stream.Score).
+
+            Args:
+                path: path to hrm annotations
+
+            Returns:
+                music21.stream.Score: score in music21 format
+
+    """
+    score = music21.converter.parse(path, format='humdrum')
```

Comment:
Possibly the same thing I mentioned in `muspy` is true for this score.

The `**harm` annotations in the score are interpreted by music21 as both annotations **and** notes.

Rendering the score as is results in the original music plus the notes implied by the annotation (which is probably not what a researcher would want). One way to address this is to remove all the annotations from the score once we know we want to store the score somewhere. To remove the annotations, something like this would suffice:

```python
rna = list(score.flat.getElementsByClass('RomanNumeral'))
score.remove(rna, recurse=True)
```

Reply:
I would hold on this until hearing from the `mirdata` maintainers, as I am not familiarized with this codebase.

Reply:
I think @napulen it is a good idea. Done 5fe814b

Reply:
ok it is a very good idea.

Reply:
done here 
546e553